opensky/anaconda-project.yml

```diff
@@ -0,0 +1,50 @@
+# To reproduce: install 'anaconda-project', then 'anaconda-project run'
+name: opensky
+description: Datashading spatially-optimized OpenSky flight trajectories
```

Comment:
Does the spatial optimization even have any meaning for this case, as a given point won't necessarily have the destination in the same partition?